gcovr/tests/excl-line/Makefile

```diff
@@ -23,6 +23,14 @@ coveralls: coverage.json
 	$(GCOVR) -a $< --coveralls coveralls.json
 
 clean:
+ifeq ($(OS),Windows_NT)
+	-cmd /C del /F /Q testcase*
+	-cmd /C del /F /Q *.gc*"
```

Comment:
this `"` can be removed (sh thinks this is an un-terminated word)

Reply:
It is a typo mistake, indeed.